cpp/ps0003/ps0003_soin_cpp.cpp

```diff
@@ -2,11 +2,15 @@
 #include<conio.h>
 using namespace std;
 int main(){
+	// prompting user for number 2/2
 	int number;
 	cin >> number;
+
+	// checking if the number is odd or even 3/3
 	if (number % 2 == 0)
-		cout << "Number is evan\n";
+		cout << "Number is even\n";
 	else
 		cout << "Number is odd\n";

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console